api/app/controllers/api/v1/position_templates_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Api::V1
+    # Controller for PositionTemplates
+    class PositionTemplatesController < ApplicationController
+
+        # GET /position_templates
+        def index
+            render json: { status: 'success', message: '', payload: position_templates_by_session }
+        end
+
+        # POST /add_position_template
+        def create
+            if not Session.exists?(id: params[:session_id])
+                render json: { status: 'error', 
+                    message: 'Invalid session_id', payload: position_templates_by_session }
+                return
+            end
+            if not params.require(:offer_template)
```

Comment:
`params.require` documentation says it raises an error if the requirement is not met, so I don't think this is the right way to use it.

Reply:
Okay. I've gotten rid of this and wrote a rescue from `ParameterMissing` instead.